pyccel/parser/parser.py

```diff
@@ -159,15 +178,15 @@ def append_son(self, son):
 
         self._sons.append(son)
 
-    def _parse_sons(self, d_parsers, verbose=False):
+    def parse_sons(self, d_parsers, verbose=False):
         """Recursive algorithm for syntax analysis on a given file and its
         dependencies.
         This function always terminates with an OrderedDict that contains parsers
         for all involved files.
         """
```

Comment:
Given that this method is no longer private, please add the "Parameters" and "Returns" sections to its docstring.

Reply:
done